apps/projects/app/components/Card/Issue.js

```diff
@@ -45,6 +56,52 @@ FlexibleDiv.propTypes = {
   children: PropTypes.node.isRequired,
 }
 
+const Bounty = ({ issue }) => {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const { workStatus, deadline, expLevel, openSubmission } = issue
+
+  let status = BOUNTY_STATUS[workStatus]
+  if (openSubmission && workStatus === 'funded') status = 'Accepting work submissions'
```

Comment:
I like that all the other status phrases are co-located in the `BOUNTY_STATUS` object. Can we move this string there, as well, and do something here like:

```js
if (openSubmission && workStatus === 'funded') status = BOUNTY_STATUS['open-submission']

Reply:
Sure. `'open-submission-funded'`.